freqtrade/edge/__init__.py

```diff
@@ -202,7 +227,7 @@ def _fill_calculable_fields(self, result: DataFrame) -> DataFrame:
         # 0.05% is 0.0005
         # fee = 0.001
 
-        stake = self.config.get('stake_amount')
+        stake = 0.015
```

Comment:
please explain ... ? this seems ... wrongish

Reply:
no it is not wrong. all things this function calculates are relative. but the stake that you want ... the is no absolute values at the end, only percentages ...

Reply:
maybe add a comment explaining this - as it's not really clear after reading just through the code.